### ISSUE TITLE
Compilation Issues - NatCorder 1.8.0

### ISSUE THREAD

**Issue by adampsst** [natsuite/NatCorder-Performance-Extensions#3]
Just updated to NatCam 1.8.0 from the asset store, added the packages to the manifest as described, and now there are many compiler errors . For example:

`Library/PackageCache/com.natsuite.ncpx@2d0fc77c7c/Runtime/Inputs/BlitCameraInput.cs(11,11): error CS0246: The type or namespace name 'Clocks' could not be found (are you missing a using directive or an assembly reference?)
`

`Library/PackageCache/com.natsuite.ncpx@2d0fc77c7c/Runtime/WEBMRecorder.cs(10,40): error CS0246: The type or namespace name 'NativeRecorder' could not be found (are you missing a using directive or an assembly reference?)
`

**Comment by olokobayusuf** [comment]
Hey @adampsst , delete NatCorder in your project (should be in `NatSuite > NatCorder` or `NatSuite > Plugins > Managed > Recorders`). Then reimport NatCorder from the Asset Store.

**Comment by adampsst** [comment]
I have done this twice today, still same issue. I just downloaded the package from the store, and I can see inside NatSuite/NatCorder/Changelog.md that the version is 1.8.0?

**Comment by adampsst** [comment]
Although checking all other Changelog files, they state different versions.

**Comment by adampsst** [comment]
Ok, looks like it is an import issue. Let me try again. I update the asset in another project and then exported. Will try again now with a fresh import.

**Comment by olokobayusuf** [comment]
@adampsst sounds good. You should [join the NatSuite community on Discord](https://discord.gg/MdwdKnDU)! It's great for small questions like this.

**Comment by adampsst** [comment]
Will do!

On 26 April 2021 at 16:54:01, Yusuf Olokoba ***@***.***)
wrote:

@adampsst <https://github.com/adampsst> sounds good. You should join the
NatSuite community on Discord <https://discord.gg/MdwdKnDU>! It's great for
small questions like this.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/natsuite/NatCorder-Performance-Extensions/issues/3#issuecomment-826949771>,
or unsubscribe
